src/cascadia/TerminalApp/TabRowControl.cpp

```diff
@@ -23,4 +33,42 @@ namespace winrt::TerminalApp::implementation
     void TabRowControl::OnNewTabButtonClick(IInspectable const&, Controls::SplitButtonClickEventArgs const&)
     {
     }
+
+    // Method Description:
+    // - Bound in Drag&Drop of the Xaml editor to the [+] button.
+    // Arguments:
+    // <unused>
+    void TabRowControl::OnNewTabButtonDrop(IInspectable const&, winrt::Windows::UI::Xaml::DragEventArgs const&)
```

Comment:
Do we have to make and bind this if nothing is going to happen? Is this some XAML trickery?

Reply:
I think so, I did copy-cat of the other Drag&Drop stuff.
I`ll try to remove it and see what happens

Reply:
Whichever way it works, I'm fine with it. It was just an interesting curiosity to me.